.changeset/mighty-poems-know.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/view-components": patch
```

Comment:
Is `patch` appropriate?

Reply:
I think we do minor now? I forget. /cc @camertron

Reply:
Yeah, I think a minor bump is appropriate here.